src/main/java/uk/co/ramp/policy/VirusStatusIsolationProperty.java

```diff
@@ -0,0 +1,12 @@
+package uk.co.ramp.policy;
+
+import org.immutables.gson.Gson;
+import org.immutables.value.Value;
+import uk.co.ramp.people.VirusStatus;
+
```

Comment:
It might be good to move some of the immutable objects into a sub-package to clear up the policy level a little.

Reply:
I prefer to structure new features (such as the isolation policy) together in their own individual packages. 
Some more info here on the concept of 'package by feature, not layer': http://www.javapractices.com/topic/TopicAction.do?Id=205#:~:text=The%20package%2Dby%2Dfeature%20style%20still%20honors%20the%20idea%20of,t%20seem%20necessary%20or%20desirable.

This allows us to make everything within the 'policy' package be either 'package private' or 'private' with the exception of the isContactIsolated() method in the IsolationPolicy interface which will be 'public'. This ensures that all implementation details of the isolation policy are encapsulated within the policy package, and not exposed unnecessarily to other packages.

Reply:
That's fair enough. I've added that to my reading list. It was a style thing, but that's ver person anyway.